src/python/model_api/visualizer/primitive/label.py

```diff
@@ -40,13 +40,13 @@ class Label(Primitive):
 
     def __init__(
         self,
-        label: str,
+        label: Union[str, float],
         fg_color: Union[str, tuple[int, int, int]] = "black",
         bg_color: Union[str, tuple[int, int, int]] = "yellow",
         font_path: Union[str, BytesIO, None] = None,
         size: int = 16,
     ) -> None:
-        self.label = label
```

Comment:
Why do we need to construct label from `float`?

Reply:
This was in case someone passes scores instead of text. One option is that the users convert the scores to string before passing it to the class. Or, we can let the class handle this. If we let the class handle this then we can introduce confidence rounding. Let me know what you prefer.